### DISCUSSION TITLE
BUG the new version  Version: 2024.02.23.1927

### DISCUSSION THREAD

**Discussion by alencarwell** [immense/Remotely#842] [General]
Hello, I installed the new version Version: 2024.02.23.1927
ON Ubuntu Server 22.04
However, I noticed that in the Download part, in Resident Agents, when we download the installer, instead of downloading Remotely_Installer.exe it downloads the Install-Remotely.ps1 script. However, as everyone knows, you cannot apply a ps1 script via GPO on a large network with more than 5 thousand PCs. Because Windows natively prohibits script execution.

The only way was to use the silent installation, but with the removal of the .EXE installer they made the use of the tool unfeasible.

But why did they do this?

They removed the .EXE but left information there on how to install it silently, but it can no longer be used.

Can someone help me?

**Comment by bitbound** [comment]
> But why did they do this?

Please see the below release notes.  I forgot to update the downloads page to reflect that it's a PowerShell script.  I'll update it on the next release.

Going forward, I highly recommend reading the release notes before installing new versions.

![image](https://github.com/immense/Remotely/assets/20995508/05115f21-d548-4841-a3f6-e013f2fed548)

**Comment by alencarwell** [comment]
OK, thanks.

**Comment by alencarwell** [comment]
Hello, I'm having difficulty applying your ps1 script to my GPO, it takes too long to load the ZIP file from the server.

And one thing I checked here was, the ps1 GPO is always executed at logon or logoff, but then the GPO will always install the agent when the user logs on to the machine.

Wouldn't it be possible to generate an MSI for the agent so that this doesn't happen?

The ps1 script is more useful for manual installation, via GPO it would be much better if it were MSI.

It is possible?
